### PR TITLE
[JENKINS-40540] Adding to the documentation that in order to run the …

### PR DESCRIPTION
**Pull request by vilacides**
…tests a Firefox 46 or older is needed.

### CODE REVIEW

File: docs/BROWSER.md [hunk 1]
```diff
@@ -24,6 +24,8 @@ Therefore, to run tests with Safari, you'd execute:
 
 See `FallbackConfig.java` for how the browser is selected.
 
+Please note that since Selenium 2.x is the version used, you will need Firefox 46 or older in order to run the tests.
+
```
Comment (andresrc) [inline docs/BROWSER.md]: ...in order to run the tests using this browser.

**Comment by raul-arabaolaza** [comment]
:bee:

**Comment by mafraba** [comment]
🐝

**Comment by olivergondza** [comment]
This info will get outdated fairly soon. How about pointing users to selenium support page instead?

**Comment by vilacides** [comment]
@olivergondza that was my intent but there is no such a page that explains compatibility between browser versions and selenium versions 1:1. The [selenium support page](http://www.seleniumhq.org/about/platforms.jsp) talks about supporting the latest ESR (and the previous to that) which is not Firefox 46 but Firefox 45.6. That doesn't match what the support page says.

**Comment by olivergondza** [comment]
I have switched to ESR a while ago as FF got broken with every other update. I do not strictly object to have this line in but it seems more helpful to say we support ESR only, as that is what selenium guys say and has a chance to save someone a ton of time.

**Comment by vilacides** [comment]
@olivergondza it took me quite some time to get the right version of Firefox to run the tests and a specific version or a link to the Selenium doc would have saved me a lot of time. That is why I did this PR and I totally agree with you that is better to limit the documentation to what Selenium guarantees that will work at any time. Change pushed!

**Pull request merged**